symfony/templates/_menu_item.html.twig

```diff
@@ -1,5 +1,14 @@
 {% set is_active = app.request.attributes.get('_route') == name %}
 
-<li class="nav-item">
-    <a class="nav-link {% if is_active %}active{% endif %}" {% if is_active %}aria-current="page"{% endif %} href="{{ path(name) }}">{{ pretty_name|raw }}</a>
-</li>
+<a
+    {% if priority is defined %}data-priority="{{ priority }}"{% endif %}
+    class="nav-link{% if is_active %} active{% endif %}"
```

Comment:
FYI (not necessarily viable over here): I've added HTML extensions, so you can use `html_classes`:
https://github.com/veelkoov/fuzzrake/blob/4ddac771e979919966a358417e6a9868152348ec/symfony/templates/main/htmx/creators_in_table.html.twig#L6
https://twig.symfony.com/doc/3.x/functions/html_classes.html

Reply:
Oh sweet! We have this in JavaScript as the [`classnames`](https://www.npmjs.com/package/classnames) package. I'm super familiar with this kind of utility and it's super helpful.

I've updated the branch to use `html_classes`.